src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  health = 100;
+  constructor(name) {
+    this.name = name;
```

Comment:
- You should allow to create animals with different health, it's only by default 100
- 100 is a magic value, create a variable